app/forms/form1095as_form.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
I'm tempted to refactor all these forms into one form, because it looks like the only thing that differs is a single string.